dso-l2/src/main/java/com/tc/objectserver/entity/EntityManagerImpl.java

```diff
@@ -57,6 +58,7 @@
 public class EntityManagerImpl implements EntityManager {
   private static final TCLogger LOGGER = TCLogging.getLogger(EntityManagerImpl.class);
   private final ConcurrentMap<EntityID, ManagedEntity> entities = new ConcurrentHashMap<>();
+  private final ConcurrentMap<Long, ManagedEntity> entityIndex = new ConcurrentHashMap<>();
```

Comment:
Since this just uses a Long as the key, you should add a comment here to describe what the key actually means.